extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -215,6 +216,14 @@ export class AmpAutocomplete extends AMP.BaseElement {
 
   /** @override */
   buildCallback() {
+    const doc = this.element.ownerDocument;
+    const isEmail = doc && isAmp4Email(doc);
```

Comment:
When will `doc` be falsy?

Reply:
It could be `null` if `this.element` is itself a document. The assertion is more for the signature of `isAmp4Email`, so we would still need a type assertion: `isAmp4Email(/** @type {!Document} */ (doc))`